apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -0,0 +1,43 @@
+<p>
+  Links are used for navigation between pages. They can be standalone links used on their own or
+  inline links placed within a sentence.
+</p>
+<p>
+  A link should be used for simple navigation. If a user triggers an action, makes a choice, or in
+  any way manipulate data, use a <a routerLink="/home/showcase/button">button</a> instead.
```

Comment:
I don't completely agree with this. I think it's a question of knowing when to use a link `<a>` and when to use a `<button>`. 

Consider making the distinction more clear e.g., by describing what links should be used for and what they should _not_ be used for. Maybe use a `<button>` instead of the `<a routerLink="/home/showcase/button">button</a>` example.

A sidenote: There's an important difference that affects accessibility (well, there is more than one 😊). Different keys are used to trigger "click link" (<kbd>enter</kbd>) and "click button" (<kbd>space</kbd>). Using links disguised as buttons can be confusing to users.

Reply:
Oh, wait. I think I read this wrong 😁. You _are_ suggesting that a `<button>` is used, right?

Reply:
Yes, I'll reverse it so it says 'A link should be used for simple navigation. Use a button if a user [reasons]'